### ISSUE TITLE
how to build a chinese to english model using t2t

### ISSUE THREAD

**Issue by yuimo** [tensorflow/tensor2tensor#431]
hi, i'm using t2t  to build my own chinese to english translation system with my own corpus
is there any suggestions for this work?
for examples, size of the corpus to train a practical system?
size of vocabulary?
more details are appreciated.
thanks a lot.

**Comment by twairball** [comment]
@yuimo You can take a look at my WMT En-Zh experiments: https://github.com/twairball/t2t_wmt_zhen

**Comment by yuimo** [comment]
@twairball thanks for your reply first.
i have read the code of problem TranslateEnzhWmt8k, have you read it too?
and i have some questions:
1)it looks like that you use seperated vocabulary, and in you hpara_set, shared_embedding_and_softmax_weights is True. and as i know, this should be False
2)somebody says that for chinese, there is no need to segment the sentences first, but in your project, you segment the sentences first with jieba
any ideas about this?

**Comment by twairball** [comment]
1) Yes it should be separate vocab for English and Chinese. So we have `vocab.zh` and `vocab.en` with different subwords

2) I would like to experiment more on this to resolve this definitively, but I don't have access to vast GPU to test many experiments. Doing segmentation first with Jieba guarantees smaller subword vocab. From experience however, I've not yet observed any noticeable difference.

TL;DR: use separate vocab, and its ok to not use jieba.

**Comment by yuimo** [comment]
yes, i'm training a ch-en translate system using the original implementary of t2t like TranslateEnzhWmt8k, and i will report the result later.
but for as the above question, should we set shared_embedding_and_softmax_weights to false when using seperated vocabulary?
